recipes/conda-pack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "conda-pack" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "d645ec1da762704a737a829346c284279e54a0e36f9ea30f8cd5192e499c045b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
```

Comment:
Is there any reason to prevent this from running on Windows?  I think it's much less necessary on Windows, but if it creates a good paradigm for moving environments around as archives, why not make that uniform across all OS's?

Reply:
It currently doesn't work on windows. There's nothing preventing that from being fixed (there's a github issue, just takes some time and attention to detail).